src/index.html

```diff
@@ -22,6 +22,437 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo__Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+      class="card"
+      data-qa="card"
+    >
+        <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="iMac"
+        />
+
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h3 class="card__product-code">Product code: 195434</h3>
+
+        <div class="card__qwe">
+          <div class="card__rate-stars">
+            <div class="card__rate-star card__rate-star--1"></div>
+            <div class="card__rate-star card__rate-star--2"></div>
+            <div class="card__rate-star card__rate-star--3"></div>
+            <div class="card__rate-star card__rate-star--4"></div>
+            <div class="card__rate-star card__rate-star--5"></div>
+          </div>
+
+          <p class="card__rate-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-item">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
```

Comment:
The `data-qa` attribute for the 'Buy' link inside the first card should be `data-qa="card-hover"` instead of `data-qa="hover"` as per the task requirements.